### ISSUE TITLE
Questions about accurancy

### ISSUE THREAD

**Issue by Kate777734** [PRIS-CV/PMG-Progressive-Multi-Granularity-Training#25]
您好！
我在尝试用FGVC-aircraft飞机数据集复现论文中的结果时，与论文中报告的92.8%存在较大差距，我最高只达到了87.3%。但是在使用CUB数据集复现时达到了与论文差不多的结果。我使用aircraft飞机数据集复现的细节如下：
1、我改写了organize_dataset.py程序来组织飞机数据集，最终按照论文的要求组织成如下格式，按照飞机数据集中自带的的variants.txt文件划分为100类：
dataset
├── train
│   ├── class_707-320
|   |      ├── 1.jpg
|   |      ├── 2.jpg
|   |      └── ...
│   ├── class_727-200
|   |      ├── 1.jpg
|   |      ├── 2.jpg
|   |      └── ...
│   └── ...
└── test
    ├── class_707-320
    |      ├── 1.jpg
    |      ├── 2.jpg
    |      └── ...
    ├── class_727-200
    |      ├── 1.jpg
    |      ├── 2.jpg
    |      └── ...
    └── ...
2、我按照飞机数据集中自带的的images_variant_test.txt和images_variant_test.txt文件来划分训练集和测试集，并将划分后的图像文件夹加载到train.py和utils.py中。
3、程序的其余部分均没有改动，仍然是batch_size=16，epoch=200。
4、我使用的显卡是 NVIDIA RTX A4000。
请问我是哪里出了问题？如果能得到您的回复，我将感激不尽！

Hello!
When I tried to reproduce the results in the paper with FGVC-aircraft dataset, there was a big gap with the 92.8% reported in the paper, and my highest was only 87.3%. However, when using CUB dataset to reproduce, the result is similar to that of the paper. I use the aircraft data set to reproduce the details as follows:
1. I rewrote the organize_dataset.py program to organize the aircraft dataset, and finally organized it into the following format according to the requirements of the paper, and divided it into 100 categories according to the variants.txt file that comes with the aircraft data set:
dataset
├── train
│ ├── class_707-320
| | ├── 1.jpg
| | ├── 2.jpg
| | └── ...
│ ├── class_727-200
| | ├── 1.jpg
| | ├── 2.jpg
| | └── ...
│ └── ...
└── test
├── class_707-320
| ├── 1.jpg
| ├── 2.jpg
| └── ...
├── class_727-200
| ├── 1.jpg
| ├── 2.jpg
| └── ...
└── ...
2. I divide the training set and test set according to the images_variant_test.txt and images_variant_test.txt files in the aircraft data set, and load the divided image folders into train.py and utils.py.
3. The rest of the program remains unchanged, and batch_size=16 and epoch=200.
4. The graphics card I use is NVIDIA RTX A4000.
Excuse me, what is wrong with me? I would appreciate it if I could get your reply!

**Comment by usha998** [comment]
> 您好! الطائرات المقاتلة FGVC قد يكون من الصعب الحصول على أفضل النتائج بنسبة 92.8% من إجمالي الناتج المحلي الإجمالي لقد حققت نسبة 87.3%. معلومات عن الطائرات. 1. قم بإنشاء موقعorganize_dataset.py للحصول على معلومات جديدة قم بتنزيل ملف 100 类: مجموعة البيانات ├── القطار │ ├── class_707-320 | | ├── 1.jpg | | ├── 2.jpg | | └── ... │ ├── class_727-200 | | ├── 1.jpg | | ├── 2.jpg | | └── ... │ └── ... └── اختبار ├── class_707-320 | ├── 1.jpg | ├── 2.jpg | └── ... ├── class_727-200 | ├── 1.jpg | ├── 2.jpg | └── ... └── ... 2. قم بتنزيل ملف images_variant_test.txt وimages_variant_ قم بتنزيل ملف test.txt وقم بتنزيله من Train.py وutils.py. 3. تم تصميم مجموعة متنوعة من المنتجات، حجم الدفعة = 16، Epoch = 200. 4، تم تصميم بطاقة NVIDIA RTX A4000 .
> 
> مرحباً! عندما حاولتُ إعادة إنتاج النتائج الواردة في البحث باستخدام مجموعة بيانات FGVC-aircraft، وجدتُ فجوةً كبيرةً مع نسبة 92.8% المذكورة فيه، وأعلى نسبة وصلتُ إليها كانت 87.3% فقط. مع ذلك، عند استخدام مجموعة بيانات CUB لإعادة الإنتاج، كانت النتيجة مشابهةً لنتيجة البحث. أستخدم مجموعة بيانات الطائرات لإعادة إنتاج التفاصيل كما يلي:
> 
> 1. لقد قمت بإعادة كتابة برنامج organize_dataset.py لتنظيم مجموعة بيانات الطائرات، وقمت في النهاية بتنظيمها بالتنسيق التالي وفقًا لمتطلبات الورقة، وقسمتها إلى 100 فئة وفقًا لملف variants.txt الذي يأتي مع مجموعة بيانات الطائرات: 
>    dataset 
>    ├── train 
>    │ ├── class_707-320 
>    | | ├── 1.jpg 
>    | | ├── 2.jpg 
>    | | └── ... 
>    │ ├── class_727-200 
>    | | ├── 1.jpg 
>    | | ├── 2.jpg 
>    | | └── ... 
>    │ └── ... 
>    └── اختبار 
>    ├── class_707-320 
>    | ├── 1.jpg 
>    | ├── 2.jpg 
>    | └── ... 
>    ├── class_727-200 
>    | ├── 1.jpg 
>    | ├── 2.jpg 
>    | └── ... 
>    └── ...
> 2. أقوم بتقسيم مجموعة التدريب ومجموعة الاختبار وفقًا لملفات images_variant_test.txt وimages_variant_test.txt في مجموعة بيانات الطائرة، وأقوم بتحميل مجلدات الصور المقسمة إلى train.py وutils.py.
> 3. يظل باقي البرنامج دون تغيير، و batch_size=16 و epoch=200.
> 4. بطاقة الرسومات التي أستخدمها هي NVIDIA RTX A4000. 
>    عفواً، ما المشكلة؟ سأكون ممتناً لو حصلت على ردك!
